src/dvclive/plots/sklearn.py

```diff
@@ -117,24 +95,18 @@ def dump(self, val, **kwargs) -> None:
 
 
 class ConfusionMatrix(SKLearnPlot):
-    DEFAULT_PROPERTIES = {
-        "template": "confusion",
-        "x": "actual",
-        "y": "predicted",
-        "title": "Confusion Matrix",
-        "x_label": "True Label",
-        "y_label": "Predicted Label",
-    }
-
-    def __init__(self, name: str, output_folder: str, **kwargs) -> None:
-        super().__init__(name, output_folder)
-        self.normalized = kwargs.get("normalized") or False
-
-    def get_properties(self):
-        properties = copy.deepcopy(self.DEFAULT_PROPERTIES)
-        if self.normalized:
-            properties["template"] = "confusion_normalized"
-        return properties
+    def __init__(self, name: str, output_folder: str, **plot_config) -> None:
+        plot_config["template"] = (
+            "confusion_normalized"
+            if plot_config.pop("normalized", None)
```

Comment:
We already released and put on docs so I kept this, but the same could now be achieved with:

```py
live.log_sklearn_plot(
    "confusion_matrix",
    [0, 0, 1, 1],
    [0, 1, 1, 0],
    name="confusion_matrix_normalized",
    template="confusion_normalized"
)
```